addons/forum-post-snip/addon.json

```diff
@@ -0,0 +1,27 @@
+{
+  "$schema": "https://raw.githubusercontent.com/ScratchAddons/manifest-schema/dist/schema.json",
+  "name": "Collapse nested forum quotes",
+  "description": "Collapses forum quotes",
+  "credits": [
+    {
+      "name": "Weredime (9gr)",
+      "link": "https://scratch.mit.edu/users/9gr/"
+    }
+  ],
+  "userscripts": [
+    {
+      "url": "userscript.js",
+      "matches": ["topics"]
+    }
+  ],
+  "settings": [
+    {
+      "name": "Post collapse",
```

Comment:
Not very descriptive

Reply:
to be honest I'm not sure what to name settings and stuff....